php/php-quiz.md

```diff
@@ -1538,3 +1538,24 @@ echo "excellent";
       2 $initialString = 'All your base are’;
       3 append($initialString);
       4 echo $initialString;
+
+#### Q106. What is the output of this script?
+```php
+$believable = 'false';
+$myth = 'The moon is made of green cheese';
+$calc = 10**3+1;
+if ($believable) {
+    echo $myth;
+}
+else {
+    echo $calc;
+}
+```
+
+- [ ] 10000
+- [ ] 31
+- [ ] 1001
+- [ ] The moon is made of green cheese
```

Comment:
I agree with your answer, however, you need to add an `x` between the brackets like ` - [x] The moon is made of green cheese`

Reply:
@haxkd can you pls take a look, thanks

Reply:
fyi i have quickly checked with chat gpt and the answer is = 31

Reply:
@chrishalbert answer already marke by another PR.

@Ebazhanov answer is "The moon is made of green cheese" after running it into my local env.